### ISSUE TITLE
Add possibility to attach attributes to useScript hook

### ISSUE THREAD

**Issue by dohomi** [charlie-tango/hooks#93]
Hello,

I came across your project and think its very useful. Sometimes it is important to attach attributes to the script tag which is being attached to the html document. I think that would be a great addition. I wanted to start a PR but I am not familiar with the mono repo structure.

Cheers

**Comment by dohomi** [comment]
This were my changes:
```js
function useScript(url?: string, attributes?: {[k:string]: string}): [boolean, ScriptStatus] {
(...)
if (!script) {
      (...)
      if(attributes){
        Object.keys(attributes).forEach(i => {
          script?.setAttribute(i, attributes[i])
        })
     } 
(...)
  }, [url, attributes])
```

**Comment by thebuilder** [comment]
Closed by #94
